### PR TITLE
Gt/add noise and flow parms

### PR DESCRIPTION
**Pull request by gregtucker**
Adds random noise to initial cone, provides parameters for flow router, and tweaks demo notebook.

### CODE REVIEW

File: src/volcanic_island_simulator.py [hunk 1]
```diff
@@ -56,14 +63,15 @@ def __init__(self, params={}):
             cone_params = _DEFAULT_CONE_PARAMS
```
Comment (vaga3461) [inline src/volcanic_island_simulator.py:63]: does this need to be indented? (or, would an unnecessary indentation cause an issue?)

**Review by vaga3461 — COMMENTED** [review]
neato!

File: src/volcanic_island_simulator.py (hunk 1 above)
Comment (gregtucker) [reply src/volcanic_island_simulator.py:63]: yes because it comes under the "else" clause

**Comment by gregtucker** [comment]
Thanks @vaga3461 I'll go ahead and merge

**Pull request merged**